### DISCUSSION TITLE
How to Withdraw the fund deposited ?How to withdraw funds from aave using python?

### DISCUSSION THREAD

**Discussion by rishab-nahar** [smartcontractkit/full-blockchain-solidity-course-py#785] [Q&A]
In Chapter 10, we see how to deposit eth(weth),borrow DAI using weth as collateral and then repaying the debt.But How are we supposed to withdraw our fund out of the Lending pool to our account...?

Looking at Lending pool.withdraw whose function definition looks as follows.
  function withdraw(
        address asset,
        uint256 amount,
        address to
    ) external returns (uint256);
we need to send address of the asset but I don't know what is the asset (weth,eth,aeth ..?),and also about how will approve_erc20 work ..?

Any help will be appreciated.

**Comment by cromewar** [comment]
Hello  @rishab-nahar , you need two things, the token you are going to use and a provider for the lending pool, both are on your brownie-config.yaml, the approve function comes directly from openzeppelin contracts and is meant to be used as a filter to allow or disallow transfers on that token.
